content/documentation/glossary/index.adoc

```diff
@@ -0,0 +1,207 @@
+---
+title: "Glossary"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a logical group of <<Workload>> defined by the <<Label App>>.
+
+=== Application Name
+
+It's the name of the <<Application>> deployed in your environment. This name is provided by the <<App Label>> on the <<Workload>>
```

Comment:
here it is called "App Label" and on line 19 it is "Label App" - this should be unified to the same name one way or another :)
Also, "App" is a name of a label and hence it should be with quotes or something like that.

Reply:
Yes, I changed to Label App I missed some links. Thanks